routes/author/index.js

```diff
@@ -19,59 +18,64 @@ module.exports = function(req, res){
       } finally {
         book.images = images;
       }
-      if(err){
+      if (err) {
         console.log(err);
         return book;
       }
       return book;
     });
-  };
+  }
 
-  var authorId = Number(req.params[0]);
-  var authors = {};
+  const authorId = Number(req.params[0]);
+  const authors = {};
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).